WordPress/src/main/res/values/strings.xml

```diff
@@ -1665,4 +1665,14 @@
 
     <string name="photo_picker_title">Choose photo</string>
 
+    <!-- E-mail verification reminder dialog -->
+    <string name="toast_saving_post_as_draft">Saving post as draft</string>
+    <string name="toast_verification_email_sent">Verification email sent, check your inbox</string>
+    <string name="toast_verification_email_send_error">Error sending verification email. Are you already verified?</string>
+    <string name="editor_confirm_email_prompt_title">Please confirm your email address</string>
+    <string name="editor_confirm_email_prompt_message">We sent you an email when you first signed up. Please open the message and click the blue button to enable publishing.</string>
+    <string name="editor_confirm_email_prompt_message_with_email">We sent an email to %s when you first signed up. Please open the message and click the blue button to enable publishing.</string>
+    <string name="editor_confirm_email_prompt_positive" translatable="false">@string/ok</string>
```

Comment:
Same as the other PR, we should drop this and use `android.R.string.ok` directly. Also I guess we could drop the "ok" string from here and use `android.R.string.ok` everywhere.